contracts/token/psp22/extensions/metadata.rs

```diff
@@ -1,13 +1,14 @@
-use brush::{
-    declare_storage_trait,
-    traits::InkStorage,
+pub use crate::{
+    psp22::*,
+    traits::psp22::extensions::metadata::*,
 };
+use brush::declare_storage_trait;
+pub use derive::PSP22MetadataStorage;
 use ink_prelude::string::String;
 use ink_storage::{
     traits::SpreadLayout,
     Lazy,
```

Comment:
I wanted to ask, we removed Lazy from nfts and multi token, why do we use it here?

Reply:
Because we forgot to do that)